languages/elixir/exercises/concept/multiple-clause-functions/.meta/design.md

```diff
@@ -0,0 +1,61 @@
+# Concept Exercise: multiple-clause-functions
+
+## Goal
+
+The goal for this exercise is to teach more about functions in elixir and how to use multiple functions clauses, guards
+
+## Learning objectives
+
+A student upon completion will:
+
+- Know what a guard is.
+- Know how to use a guard.
+- Know that guards return strict boolean values.
+- Know that guards meet certain guarantees to not mutate state.
+- Know that guards must be declared in a special form, and not all functions that return boolean values are guards.
+- Know how to write basic multiple-clause functions.
+- Know how to use guards to protect execution of a function.
```

Comment:
I might be unfamiliar with some vocabulary here, what does it mean to "protect execution of a function"? Prevent?

Reply:
maybe 'selectively execute functions` is better?